README.md

```diff
@@ -1,55 +1,166 @@
 
-## Install
-1. Setup the [web server](/server) to serve the desired OTA binaries to your device.
-2. Build the [ESP-IDF binary](/espressif) or download the Release binary to flash the device.
-3. Trigger an update via the product's app.
-4. Connect to the Tasmota hotspot and configure!
+![nrf2](https://github.com/kubedzero/switchbota/assets/51148401/c3f5a073-bc23-4cf2-a2f8-a926bfd45af7)
 
-## Tasmota Setup
-Currently, Tasmota's support for the ESP32-C3 is unofficial. However, power monitoring and MQTT delivery works effectively with the following template:
+- Select the BYTE_ARRAY type from the dropdown and enter in the hex code `57 0F 0A 01 0C` (without spaces). Then it Send to transmit the command to the plug
+  - The last two characters represent the firmware version to be flashed, and must be different from the current firmware version. `0C` (as shown above) represents v1.2, `0D` represents v1.3, `0E` represents v1.4. `0C` should be safe to use by default. More detail can be found [here](https://github.com/kendallgoto/switchbota/issues/3#issuecomment-1491004013) 
 
-```json
-{"NAME":"W1901400","GPIO":[0,0,32,0,0,0,224,320,321,0,0,0,0,0,0,0,0,0,2720,2656,2624,0],"FLAG":0,"BASE":1}
-```
-[More Template information](https://templates.blakadder.com/switchbot_plugmini_W1901400.html)
+![nrf3](https://github.com/kubedzero/switchbota/assets/51148401/93533e16-bc48-40b6-90af-c89f98381f54)
 
-[How to use a template?](https://templates.blakadder.com/howto.html)
+- Wait a few seconds and look at the NodeJS log to verify the command worked as it should. You should see something such as `::ffff:192.168.0.131 - /version/wocaotech/firmware/WoPlugUS/WoPlugUS_V12.bin` which indicates the NodeJS server was able to send the first `.bin` file to the plug
+- Wait about a minute after seeing the NodeJS server log to give the plug time to get ready for the next step.
+- Now send a new BYTE_ARRAY command in the BLE app (nRF Connect) with hex `57 0F 0B`. Hit Send to trigger the second and final firmware flash step
+- Wait a few seconds again and then look at the NodeJS log to verify the command worked as it should. You should see something such as `::ffff:192.168.0.131 - /payload.bin` which indicates the NodeJS server was able to send the Tasmota `.bin` file to the plug
+- DONE! The firmware flashing should be complete
 
-## Process
+## Part 4: Tasmota Setup
+
+- **TL;DR: Connect to the Tasmota WiFI. Configure the actual WiFi. Update the firmware. Apply the SwitchBot Tasmota Template. Configure the device**
+- NOTE: The NodeJS server is configured to flash Tasmota `tasmota32c3.factory.bin` which is described to be a "Factory binary to be used for initial flashing using esptool." A more fully-featured firmware should be flashed immediately after.
```

Comment:
nit: `factory` payloads aren't (usually) inherently less-featured. they might be in this specific case, since the c3 binary we're linked against is quite old. however, the distinguishing factor is that a factory binary includes an entire partition table, bootloader, etc., rather than just application partition that is later sent for OTA binaries in the future.